services/web/server/src/simcore_service_webserver/projects/_db_utils.py

```diff
@@ -275,7 +275,26 @@ async def _get_project(
 
         query = (
             sa.select(
-                *[col for col in projects.columns if col.name not in ["access_rights"]],
+                projects.c.id,
+                projects.c.type,
+                projects.c.uuid,
+                projects.c.name,
+                projects.c.description,
+                projects.c.thumbnail,
+                projects.c.prj_owner,  # == user.id (who created)
+                projects.c.creation_date,
+                projects.c.last_change_date,
+                projects.c.workbench,
+                projects.c.ui,
+                projects.c.classifiers,
+                projects.c.dev,
+                projects.c.quality,
+                projects.c.published,
+                projects.c.hidden,
+                projects.c.trashed,
+                projects.c.trashed_by,  # == user.id (who trashed)
+                projects.c.trashed_explicitly,
+                projects.c.workspace_id,
```

Comment:
tbh I really do not like this approach and really do not get what is the advantage of doing it like so? I see only more maintenance work.

Reply:
there is a ground reason. Had discussion with @matusdrobuliak66 and @giancarloromeo . I will be building an example to drive the design conventions that justify this